src/lib/viz/layer/Layer.ts

```diff
@@ -261,11 +254,11 @@ export class Layer extends WithEvents implements StyledLayer {
   /**
    * Method to create the Deck.gl layer
    */
-  public async _createDeckGLLayer(forceInit = false) {
+  public async _createDeckGLLayer() {
```

Comment:
Should we rename this method, removing the underscore, as this is public?

Reply:
Yes, we need to follow always the same convention (I prefer not to use underscore with TS). So, I will change it